packages/product-react/src/components/Container/Container.tsx

```diff
@@ -0,0 +1,68 @@
+import clsx from 'clsx';
+import React, { ReactHTML, ReactNode } from 'react';
+
+export type ContainerProps = {
+  align?: 'left' | 'right';
```

Comment:
Can we update this and the other string literal type (`justify`) to something like this?
```
export enum AlignNameHere {
  LEFT = 'small',
  RIGHT = 'right',
}
...
align?: keyof typeof AlignNameHere;
```